.github/CODE_OF_CONDUCT.md

```diff
@@ -28,6 +28,7 @@ Examples of unacceptable behavior by participants include:
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic
  address, without explicit permission
+* Making pull requests or raising issues for personal benefit or harassment
```

Comment:
Could you be more precise of what is your intent behind that?
Harassment is already covered before.

Reply:
@Jainan-Tandel Any feedback?

Reply:
No feedback and seems not relevant.